### ISSUE TITLE
Approach for pages that do not exist

### ISSUE THREAD

**Issue by bjohansebas** [expressjs/expressjs.com#1686]
I open this issue to see if everyone or the majority agrees on redirecting the links of the pages that do not exist to English, as done in #1685. I'm asking this before launching that PR

**Comment by chrisdel101** [comment]
I like having an approach. My _personal_ opinion is that English links should not be in other languages since it's hard to get back to the language (have to use back button). BUT I'm okay with this (English links) if that's the consensus. Better to be consistent :)
(And English is better than broken)

**Comment by bjohansebas** [comment]
That's true, English is better than broken English, but this would be fixed in the future if translation automation is achieved.

**Comment by carlosstenzel** [comment]
I agree, since we will have automatic translation, it is better to have one updated page in English, than several outdated pages.

**Comment by carlosstenzel** [comment]
What do you think about having an icon for a translation page in the navbar?

![Captura de Tela 2024-11-19 às 21 27 19](https://github.com/user-attachments/assets/fed7d404-acb4-43de-a866-a483f4fd9462)
ref https://react.dev/

**Comment by peterdanwan** [comment]
Hello all - I'm happy to implement any approach that is decided! 

Just wanted to note that consistent link patterns might make it easier to `Ctrl + F` for future updates, since the links I added differ from the existing style of links.

![image](https://github.com/user-attachments/assets/c3c325d2-5808-46a4-8a42-c21e34f6410c)

**Comment by bjohansebas** [comment]
> What do you think about having an icon for a translation page in the navbar?

I think it's great! i was planning to do it soon, which is why i also opened the issue. i was thinking of redirecting the page the user is on to their language, but this could lead to several 404 errors since there are pages that haven’t been implemented yet.

> Just wanted to note that consistent link patterns might make it easier to Ctrl + F for future updates, since the links I added differ from the existing style of links.

Yep, that's why, among other reasons, we should reach a consensus on this. Maybe @jonchurch and @crandmck can give us their opinion.

**Comment by bjohansebas** [comment]
The only link I think we should have in English is the blog (I mean for the other languages). For the rest, we can add the page as they are reported. This way, we won’t forget over time, and in the future, when automation is in place, we won’t have to worry about this.

**Comment by chrisdel101** [comment]
Then do we want #1685 to just remove those links rather replace them?

**Comment by carlosstenzel** [comment]
I don't think removing it is a good practice

**Comment by bjohansebas** [comment]
> Then do we want https://github.com/expressjs/expressjs.com/pull/1685 to just remove those links rather replace them?

Adding those pages is what I mean

**Comment by chrisdel101** [comment]
I don't get it. Can you elaborate?

**Comment by bjohansebas** [comment]
I mean adding the file with the content in English. That is, if `/es/guide/migration` does not exist, the file should be added to avoid generating a 404.

Therefore, we should not change the link in #1685 but rather add the content. The same applies to #1615, we should add the support pages.

The only content that cannot be translated (yet) is the blog, so in other languages, it makes sense for the link to remain in English.

<details><summary>My message in Spanish</summary>
<p>

Me refiero a agregar el archivo con el contenido en inglés. Es decir, si `/es/guide/migration` no existe, se debe agregar el archivo para evitar que se genere un error 404,

Por lo tanto, no deberíamos cambiar el enlace en el #1685, sino más bien agregar el contenido. Lo mismo aplica para el #1615: deberíamos agregar las páginas de soporte.

El único contenido que no se puede traducir (todavía) es el blog, por lo que en otros idiomas tiene sentido que el enlace permanezca en inglés.
</p>
</details>

**Comment by carlosstenzel** [comment]
@bjohansebas 
The main objective of the issue (page 404) has already been created, could we close it?
